future/src/ct/ct_storage_control.h

```diff
@@ -58,8 +58,31 @@ class CtStorageControl
     void pending_new_db_node(gint64 node_id);
     void pending_rm_db_nodes(const std::vector<gint64>& node_ids);
     void pending_edit_db_bookmarks();
-private:
+    
+    constexpr CtStorageEntity* get_storage() const { return _storage; }
+    constexpr void set_storage(CtStorageEntity* storage) {
+        delete _storage;
```

Comment:
could we use a std::unique_ptr here and .reset()?

Reply:
I see no reason for get_storage and set_storage at all